### ISSUE TITLE
additional resources - dead link on home page

### ISSUE THREAD

**Issue by ilessing** [UCSantaBarbara/webguide#115]
see: http://webguide.ucsb.edu/#additional-resources

Link Text: Web Standards Guide Resources
Dead link: https://it.ucsb.edu/resources/web-standards-guide-resources

**Comment by garster** [comment]
Should this link to the wiki now?
https://github.com/ucsb-wsg/ucsb-wsg.github.io/wiki

**Comment by loganfranken** [comment]
I think this is up for debate; maybe we want to discuss it in a meeting?

Per #68, there was a desire to leave the "Website Tools" section on the IT resources website:
https://it.ucsb.edu/resources/website-tools

Given that, I imagine the group may want both a (1) link to that remaining IT page and (2) a link to the GitHub wiki?

**Comment by fablio-work** [comment]
I think it’s probably worth discussing.  A topic related to this was brought up at this past WSG meeting.

That had to do with moving meeting scheduling/event pages to another service (Google Groups???).
This was brought up because one of the WSG chairs has had a simple page edit crash the 
it.ucsb.edu http://it.ucsb.edu/ site (I believe I captured that correctly)…  I have seen more issues with editing oddities
(I like using the list elements, and hitting an enter causes inconsistent behavior with the cursor location,
sometimes it works correctly, sometimes the cursor jumps to another location in the edit window).

With some of these behaviors  and the support situation for the it.ucsb.edu http://it.ucsb.edu/ site “evolving", my 
feeling is that we (WSG and WTX) should be looking more aggressively at alternatives.

To recap the concerns with incorporating the website tools information into the WSG:
- avoid any perceived UCSB endorsement of a particular product
- in our rapidly evolving toolscape - the content can get stale relatively quickly
- to limit candid discussion of pros and cons of a particular product to the campus (although I am not sure this actually ever happened)

Aaron Martin  (CNT)
Earth Research Institute
6809 Ellison Hall
UCSB MS-3060
805-893-8415
aaron@eri.ucsb.edu mailto:aaron@eri.ucsb.edu

**Comment by loganfranken** [comment]
@garster @UCSB-jcolo Could we tag this for discussion at the next meeting? Thanks!

**Comment by UCSB-jcolo** [comment]
Sounds like a plan.
I have it slated for the next meeting.
Thanks guys

**Comment by loganfranken** [comment]
Hey there! Did this end up getting discussed?

**Comment by tenken** [comment]
I believe indirectly.

We talked about where some missing content should be moved too, but didnt come up with an answer as to whether the link should stay; both "wiki" links to it.ucsb.edu and the github wiki should be shown on the page or other options ...

To recap the issue some of the content in the it.ucsb.edu is UCSB specific resources, which could appear to endorse a product or service due to campus adoption. While the resources we list on the github wiki are basically completely free and dont come across as an endorsement.

**I would recommend linking both URLs with a description to the purpose for each link.**

If I'm mistaken somebody please correct me. Thanks.
